tests/unit/test_mysql.py

```diff
@@ -256,7 +256,11 @@ def test_delete_users_for_unit(
 
         self.mysql.delete_users_for_unit("app/0")
 
-        _run_mysqlcli_script.assert_called_once_with(_expected_get_unit_user_commands)
+        _run_mysqlcli_script.assert_called_once_with(
+            _expected_get_unit_user_commands,
+            user="serverconfig",
+            password="serverconfigpassword",
```

Comment:
Hardcoded strings are copy&pasted a lot in codebase... we need to start using variables/constants/labels.